source/guides/guides/network-requests.md

```diff
@@ -336,7 +336,7 @@ cy.wait('@new-user')
 // we can grab the completed XHR object again to run more assertions
 // using cy.get(<alias>)
 cy.get('@new-user') // yields the same XHR object
-  .its('request.body')
+  .its('requestBody')
```

Comment:
this is not a mistake. The XHR object has `request`, `response` AND `requestBody`, `requestHeaders`, `responseBody`, `responseHeaders` properties. See https://github.com/cypress-io/cypress-example-recipes/blob/master/examples/server-communication__xhr-assertions/cypress/integration/spec.js#L111 for `request.body` access and screenshot https://github.com/cypress-io/cypress-example-recipes/tree/master/examples/server-communication__xhr-assertions#cywait-vs-cyget-for-network-requests 

This is why I put `cy.get('@new-user').then(console.log)` as a tip - so people see the real options

Reply:
Then our typings are not correct in Cypress. 

![Screen Shot 2019-12-23 at 9 58 10 AM](https://user-images.githubusercontent.com/1271364/71334023-c039dc80-256a-11ea-81ea-5d7500d83fc0.png)